src/data/nat/parity.lean

```diff
@@ -183,6 +183,13 @@ begin
     apply even_mul_succ_self }
 end
 
+lemma even_prime_eq_two {p : ℕ} (he : even p) (hp : prime p) : p = 2 :=
+begin
+  symmetry,
+  rw ←(nat.prime_dvd_prime_iff_eq prime_two hp),
+  exact even_iff_two_dvd.mp he,
+end
```

Comment:
```suggestion
((prime_dvd_prime_iff_eq prime_two hp).mp (even_iff_two_dvd.mp he)).symm
```

Reply:
In fact, why not prove the `iff` version?
```
lemma prime.even_iff {p : ℕ} (hp : nat.prime p) : even p ↔ p = 2 :=
by rw [even_iff_two_dvd, prime_dvd_prime_iff_eq prime_two hp, eq_comm]
```

Reply:
added!